Automattic/jetpack
# Instant Search: fix button styling in Twenty Twenty One theme

Fixes https://github.com/Automattic/jetpack/issues/25407.

Broken out from https://github.com/Automattic/jetpack/pull/25414 to specifically fix the sort button styling.

#### Changes proposed in this Pull Request:

Fix the display of sort buttons in Instant Search. Some themes (such as Twenty Twenty One) override our styles using very specific selectors.

#### Other information:


#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. On a site with a Jetpack Search plan, enable the Twenty Twenty One theme.
2. Open the Overlay by appending `?s=&category=uncategorized` to your site address.
3. Confirm the buttons look correct in both Light & Dark themes.
4. Toggle the light/dark themes from `/wp-admin/admin.php?page=jetpack-search-configure`.
